src/service/poc_mobile.proto

```diff
@@ -143,15 +145,86 @@ message radio_usage_stats_req_v1 {
   repeated radio_usage_carrier_transfer_info carrier_transfer_info = 13;
 }
 
-message radio_usage_stats_res_v1 { string id = 1; }
+message radio_usage_stats_res_v1 {
+  option deprecated = true;
+  string id = 1;
+}
 
 message radio_usage_stats_ingest_report_v1 {
+  option deprecated = true;
   // Timestamp in milliseconds since unix epoch
   uint64 received_timestamp = 1;
   // the verified report
   radio_usage_stats_req_v1 report = 2;
 }
 
+message radio_usage_sampling_carrier_data_transfer_info_v1 {
+  carrier_id_v2 carrier_id = 1;
+  uint64 bytes_transferred_download = 2;
+  uint64 bytes_transferred_upload = 3;
+  uint64 bytes_transferred_total = 4;
+  uint64 user_count = 5;
+}
+
+message radio_usage_carrier_data_transfer_info_v2 {
+  carrier_id_v2 carrier_id = 1;
+
+  uint64 user_count = 2;
+
+  uint64 rewarded_bytes_transferred_download = 3;
```

Comment:
looking through the data transfer sessions, I can't determine download and upload for rewarded vs unrewarded.  We get upload_bytes, download_bytes, and  rewardable_bytes, so I can't determine which of the upload vs download was rewarded.  I suggest we remove the update/download fields and just go with totals.